pkgs/top-level/config.nix

```
@@ -137,6 +137,17 @@ let
       '';
     };
 
+    allowVariants = mkOption {
+      type = types.bool;
+      default = true;
+      description = ''
+        Whether to expose the nixpkgs variants.
+
+        Variants are instances of the current nixpkgs instance with different stdenvs or other applied options.
+        This allows for using different toolchains, libc's, or global build changes across nixpkgs.
```

```suggestion
        This allows for using different toolchains, libcs, or global build changes across nixpkgs.
```

And maybe it would be good to explain why you'd want to enable/disable this?

---

Done